README.md

```diff
@@ -8,7 +8,7 @@
 
 # kubefirst Docs
 
-Welcome to the open source repository that powers [the Kubefirst documentation](https://kubefirst.konstruct.io/docs).
+Welcome to the open source repository that powers [the Kubefirst documentation](https://kubefirst.konstruect.io/docs).
```

Comment:
```suggestion
Welcome to the open source repository that powers [the Kubefirst documentation](https://kubefirst.konstruct.io/docs).
```

Reply:
My last PR was mentionning "test with bad link in MD"

Please don't review this PR until it's ready. It needs to not be draft for my tests.